apps/coscrad-frontend/src/components/higher-order-components/aggregate-detail-container.tsx

```diff
@@ -11,7 +11,7 @@ export interface AggregateDetailContainerProps<T> {
      * presenters (e.g. thumnail, maybe someday mobile) while reusing all of the
      * non-presentational logic.
      */
-    detailPresenterFactory: IResourceDetailPresenterFactory<T>;
+    detailPresenterFactory: ICategorizableDetailPresenterFactory<T>;
```

Comment:
Be sure to change all filenames as well! Don't leave any leftover `resource` utilities that should be for `categorizables`.

Reply:
Doesn't apply to this file.